### ISSUE TITLE
Renames my exe and breaks frosty mod

### ISSUE THREAD

**Issue by bdodroid** [Dyvinia/DatapathFixPlugin#3]
Took me a minute to track this down but it seems installing this plugin in frosty mod and launching the game, it renames my exe file from 
DragonAgeInquisition.exe to DragonAgeInquisition.orig.exe and then throws this error:
```
The system cannot find the file specified

   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at Frosty.ModSupport.FrostyModExecutor.ExecuteProcess(String processName, String args, Boolean waitForExit, Boolean asAdmin, Dictionary`2 env)
   at Frosty.ModSupport.FrostyModExecutor.Run(FileSystem inFs, CancellationToken cancelToken, ILogger inLogger, String rootPath, String modPackName, String additionalArgs, String[] modPaths)
   at FrostyModManager.MainWindow.<>c__DisplayClass13_0.<launchButton_Click>b__0(FrostyTaskWindow task)
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Frosty.Core.Windows.FrostyTaskWindow.<FrostyTaskWindow_Loaded>d__8.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```
   
 WIthout this plugin mod frosty launched dragon age origins but mods fail to load at all. 
  
  Trying to add the changed exe this plugin generates in frosty just throws a "There was an error when trying to load game using specified profile"
  
  I have the steam version of Dragon Age Inquisition installed to a second drive.

**Comment by Dyvinia** [comment]
what version of the plugin are you using?

**Comment by Dyvinia** [comment]
fixed with 1.4.1
